openslides/users/static/js/users/site.js

```diff
@@ -782,20 +796,94 @@ angular.module('OpenSlidesApp.users.site', ['OpenSlidesApp.users'])
                 } else {
                     user.is_committee = false;
                 }
+
+                // Check for duplicates
+                user.duplicate = false;
+                users.forEach(function(user_) {
+                    if (user_.first_name == user.first_name &&
+                        user_.last_name == user.last_name &&
+                        user_.structure_level == user.structure_level) {
+                        if (user.duplicate) {
+                            // there are multiple duplicates!
+                            user.duplicate_info += '\n' + gettext('There are more than one duplicates of this user!');
+                        } else {
+                            user.duplicate = true;
+                            user.duplicateAction = $scope.duplicateActions[0];
+                            user.duplicate_info = '';
+                            if (user_.title)
+                                user.duplicate_info += user_.title + ' ';
+                            if (user_.first_name)
+                                user.duplicate_info += user_.first_name;
+                            if (user_.first_name && user_.last_name)
+                                user.duplicate_info += ' ';
+                            if (user_.last_name)
+                                user.duplicate_info += user_.last_name;
+                            user.duplicate_info += ' (';
+                            if (user_.number)
+                                user.duplicate_info += gettext('number') + ': ' + user_.number + ', ';
+                            if (user_.structure_level)
+                                user.duplicate_info += gettext('structure level') + ': ' + user_.structure_level + ', ';
+                            user.duplicate_info += gettext('username') + ': ' + user_.username + ') '+
+                                gettext('exists already.');
```

Comment:
typo: "already exists."
You have to use "gettextCatalog.getString()" to translate the concatenated string parts.